CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+#### 0.3.2
+  * Work around an issue running Dart analyzer on the generated code, if the
+    `dynamic` type appeared in the output. Smoke will now use `Object` instead.
```

Comment:
Just to clarify, I suppose we plan to revert this once the analyzer fix lands, right?